docs/api/web-contents.md

```diff
@@ -1225,6 +1225,30 @@ Returns `Promise<NativeImage>` - Resolves with a [NativeImage](native-image.md)
 
 Captures a snapshot of the page within `rect`. Omitting `rect` will capture the whole visible page.
 
+#### `contents.isBeingCaptured()`
+
+Returns `Boolean` - Whether this page is being captured. It returns true when the capturer count
+is large then 0.
+
+#### `contents.incrementCapturerCount()`
+
+* `size` [Size](structures/size.md) (optional) - The perferred size for the capturer.
+* `stay_hidden` Boolean (optional) -  Keep the page hidden instead of visible.
+
+Increase the capturer count by one. The page is considered visible when its browser window is
+hidden and the capture count is non-zero. If you want the page stay hidden instead you should
+set the `stay_hidden` to true.
+
+This also affects the Page Visibility API.
+
+#### `contents.decrementCapturerCount()`
```

Comment:
should be `contents.decrementCapturerCount([size])`

Reply:
Fixed.